### ISSUE TITLE
Problem matching aliases on Mzalendo

### ISSUE THREAD

**Issue by GeorgieBurr** [mysociety/pombola#2501]
1) Mzalendo staff can't update the aliases, as the changes Derik is making are not being reflected as captured in the photo below. After an update, an unassigned entry still appears on the unassigned list.

![inkedaliasing on mzalendo_li](https://user-images.githubusercontent.com/38856756/47665633-712c2d80-db99-11e8-872b-5c281fac9708.jpg)

2)The intern keeps getting an error message. See attached pic.
![error message](https://user-images.githubusercontent.com/38856756/47665577-55c12280-db99-11e8-8a7e-f33784b1b5af.png)

**Comment by chrismytton** [comment]
Further information about this from the Mzalendo team:

- Aliases are entries from the hansard that did not automatically synchronise with our system as per the scrapper instructions. 
- We usually have to manually match with the full MPs name where applicable. We ignore entries not linked to a particular name. When Derrick tried to match the entries with the correct MP names the action was not updated on the system. Consequently, he could see the entries still on the unassigned entries.
- **If we don't match entries with MPs names it means their profiles are not updated.**
- The error message comes up immediately I click the next unassigned alias. This is happening for me too. Our default browser is usually Chrome.

**Comment by chrismytton** [comment]
Looks like this error might be related:

```
Internal Server Error: /admin/hansard/alias/next_unassigned/
Traceback (most recent call last):
  File "/data/vhost/mzalendo.mysociety.org/pombola-virtualenv/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/data/vhost/mzalendo.mysociety.org/pombola-virtualenv/lib/python2.7/site-packages/django/utils/decorators.py", line 34, in _wrapper
    return bound_func(*args, **kwargs)
  File "/data/vhost/mzalendo.mysociety.org/pombola-virtualenv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 22, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/data/vhost/mzalendo.mysociety.org/pombola-virtualenv/lib/python2.7/site-packages/django/utils/decorators.py", line 30, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/data/vhost/mzalendo.mysociety.org/pombola/pombola/hansard/admin.py", line 117, in next_unassigned
    context_instance=RequestContext(request)
  File "/data/vhost/mzalendo.mysociety.org/pombola-virtualenv/lib/python2.7/site-packages/django/shortcuts.py", line 45, in render_to_response
    using=using)
  File "/data/vhost/mzalendo.mysociety.org/pombola-virtualenv/lib/python2.7/site-packages/django/template/loader.py", line 137, in render_to_string
    raise TemplateDoesNotExist(template_name)
TemplateDoesNotExist: admin/hansard/alias
```

https://groups.google.com/a/mysociety.org/d/msg/cron-mzalendo/V3W-LFQk0xY/tiMyoFogAgAJ

**Comment by chrismytton** [comment]
Pull request to fix this: #2501
